.github/sync-files.yaml

```diff
@@ -0,0 +1,17 @@
+- repository: autowarefoundation/autoware
+  files:
+    - source: .github/dependabot.yaml
+    - source: .github/workflows/pre-commit.yaml
+    - source: .github/workflows/pre-commit-optional.yaml
+    - source: .github/workflows/semantic-pull-request.yaml
+    - source: .github/workflows/spell-check-differential.yaml
+    - source: .markdown-link-check.json
+    - source: .markdownlint.yaml
+    - source: .pre-commit-config-optional.yaml
+    - source: .prettierignore
+    - source: .prettierrc.yaml
+    - source: .yamllint.yaml
+
+- repository: autowarefoundation/autoware_common
+  files:
+    - source: .github/workflows/sync-files.yaml
```

Comment:
I'll add `sync-files.yaml` to autowarefoundation/autoware as well to keep this settings simple.

Reply:
https://github.com/autowarefoundation/autoware/pull/110